demo/api/schema.gql

```diff
@@ -534,6 +535,7 @@ input BrevoContactFilterAttributesInput {
 input EmailCampaignContentScopeInput {
   domain: String!
   language: String!
+  test: String!
```

Comment:
The test should be removed

Reply:
87d88415598aaa5a8682558906b7e37b972850aa